core/src/main/scala/org/apache/spark/sql/rapids/tool/ui/ConsoleProgressBar.scala

```diff
@@ -132,6 +132,23 @@ class ConsoleProgressBar(
     totalCounter.incrementAndGet()
   }
 
+  def reportSuccessfulProcesses(n: Int): Unit = {
+    for(_ <- 1 to n) {
+      reportSuccessfulProcess()
+    }
```

Comment:
Nit: You could use foreach instead of for here( It looks more of functional than imperative)
```
def reportSuccessfulProcesses(n: Int): Unit = {
  (1 to n).foreach(_ => reportSuccessfulProcess())
}
```

Reply:
Fixed